examples/6field-simple/elm_6f.cxx

```diff
@@ -383,45 +383,62 @@ int physics_init(bool restarting) {
   auto globalOptions = Options::root();
   auto options = globalOptions["highbeta"];
 
-  OPTION(options, n0_fake_prof, false); // use the hyperbolic profile of n0. If both  n0_fake_prof and
-                 // T0_fake_prof are false, use the profiles from grid file
-  OPTION(options, n0_height, 0.4); // the total height of profile of N0, in percentage of Ni_x
-  OPTION(options, n0_ave, 0.01);  // the center or average of N0, in percentage of Ni_x
-  OPTION(options, n0_width, 0.1); // the width of the gradient of N0,in percentage of x
-  OPTION(options, n0_center, 0.633); // the grid number of the center of N0, in percentage of x
-  OPTION(options, n0_bottom_x, 0.81); // the start of flat region of N0 on SOL side, in percentage of x
-  OPTION(options, T0_fake_prof, false);
-  OPTION(options, Tconst, -1.0); // the amplitude of constant temperature, in percentage
-
-  OPTION(options, experiment_Er, false);
-
-  OPTION(options, laplace_alpha, 1.0); // test parameter for the cross term of invert Lapalace
-  OPTION(options, Low_limit, 1.0e-10); // limit the negative value of total quantities
-  OPTION(options, q95_input, 5.0); // input q95 as a constant, if <0 use profile from grid
-  OPTION(options, local_q, false); // using magnetic field to calculate q profile
-  OPTION(options, q_alpha, 1.0); // flux-limiting coefficient, typical value is [0.03, 3]
-
-  OPTION(options, gamma_i_BC, -1.0); // sheath energy transmission factor for ion
-  OPTION(options, gamma_e_BC, -1.0); // sheath energy transmission factor for electron
-  OPTION(options, Sheath_width, 1);  // Sheath boundary width in grid number
-
-  OPTION(options, density, 1.0e19);   // Number density [m^-3]
-  OPTION(options, Zi, 1);             // ion charge number
-  OPTION(options, continuity, false); // use continuity equation
-
-  OPTION(options, evolve_jpar, false);    // If true, evolve J raher than Psi
-  OPTION(options, phi_constraint, false); // Use solver constraint for phi
+  n0_fake_prof = options["n0_fake_prof"].withDefault(
+      false); // use the hyperbolic profile of n0. If both  n0_fake_prof and
```

Comment:
Clang-format usually handles these a bit better if you move the comment to the previous line